### PR TITLE
Moves buttons from navigator and navigationBar into separate components

### LINKED ISSUE #10354: Split navigator in smaller components
## Test plan

https://github.com/brave/browser-laptop/pull/10355#issue-248977058

----

**Describe the issue you encountered:**
Split navigator and navigationBar into:
- bookmarkButton
- homeButton
- reloadButton
- stopButton
- forwardButton
- backButton

for #9283

### PR DESCRIPTION
**Pull request by NejcZdovc**
Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

Resolves #10354
Addresses #9283

Auditors: @cezaraugusto @bsclifton

Test Plan:
- check if reload button is working
- check if stop button is working
- check if forward/back buttons are working
- check if long press forward/back buttons are working
- check if bookmark button is working (add/remove)

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

**Comment by luixxiul** [comment]
please refer to the original commit in the commit message if you don't mind. thanks.

File: app/renderer/components/navigation/buttons/bookmarkButton.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      {
+        !this.props.titleMode
+          ? <span className='bookmarkButtonContainer'>
+            <button data-l10n-id={this.props.bookmarkKey ? 'removeBookmarkButton' : 'addBookmarkButton'}
+              className={cx({
+                navigationButton: true,
+                bookmarkButton: true,
+                removeBookmarkButton: !!this.props.bookmarkKey,
+                withHomeButton: getSetting(settings.SHOW_HOME_BUTTON),
+                normalizeButton: true
```
Comment (luixxiul) [inline app/renderer/components/navigation/buttons/bookmarkButton.js]: will this be replaced with `<NormalizedButton>` later?

**Comment by NejcZdovc** [comment]
@luixxiul which one is original commit? this is just a pre-work for your PR

File: app/renderer/components/navigation/buttons/bookmarkButton.js (hunk 1 above)
Comment (NejcZdovc) [inline app/renderer/components/navigation/buttons/bookmarkButton.js]: yeah this PR just moves things around and don't change any functionalities or introduce new ones

File: app/renderer/components/navigation/buttons/reloadButton.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      ipc.emit(messages.SHORTCUT_ACTIVE_FRAME_RELOAD)
+    }
+  }
+
+  onReloadLongPress (target) {
+    contextMenus.onReloadContextMenu(target)
+  }
+
+  render () {
+    return <span className='navigationButtonContainer'>
```
Comment (luixxiul) [inline app/renderer/components/navigation/buttons/reloadButton.js]: there was a wrapper to replace this. please see the abandoned PR. thanks.

File: app/renderer/components/navigation/buttons/bookmarkButton.js (hunk 1 above)
Comment (luixxiul) [inline app/renderer/components/navigation/buttons/bookmarkButton.js]: ok understood.

**Comment by luixxiul** [comment]
> @luixxiul which one is original commit? this is just a pre-work for your PR

ok nvm.

**Pull request merged**